docs/GettingStartedSynthDet.md

```diff
@@ -33,7 +36,7 @@ Once the data is generated locally, you can use`datasetinsights`  to show datase
 
 ```docker run -p 8888:8888 -v "<Synthetic Data File Path>":/data -t unitytechnologies/datasetinsights:latest```
 
-Replace `<Synthetic Data File Path>` with the path to the local datasets (listed in step 2.3).
+Replace `<Synthetic Data File Path>` with the path to the local datasets (listed in step 2.3). You may need to specify an absolute file path instead of a relative path for the `<Synthetic Data File Path>`.
```

Comment:
```suggestion
Replace `<Synthetic Data File Path>` with the path to the local datasets (listed in step 3.3).
```

Since the paths in 3.3 are not relative, the added sentence might be confusing. Did you mean spelling out the path completely instead of using env variables?

Reply:
Agreed, I think now that I read over it, it should be implied that its absolute